### ISSUE TITLE
Plot window focus

### ISSUE THREAD

**Issue by mducle** [mantidproject/mslice#122]
After clicking `Plot` in the `Cut` tab, the focus should change to the plot window.

Presently, if a new plot window is create it inherits focus, but if the window already exists and the data is overplotted or replotted, then it does not get focus.

Ideally, this behaviour should be modifiable by an option somewhere.

**Comment by mducle** [comment]
Can manually raise the window using:

```
import mslice.cli
import sys
window = mslice.cli.gcf().canvas.parent()
if 'Linux' in sys.platform.platform():
    window.showMaximised()
    window.showNormal()
window.raise_()
window.activateWindow()
```

The `showMaximised()` and `showNormal()` calls seem to be needed on RHEL7 (not tested on Ubuntu), but not on Windows (not tested on OSX).

**Comment by mducle** [comment]
Resolved by #134
